lib/ash/domain/info.ex

```diff
@@ -300,6 +300,27 @@ defmodule Ash.Domain.Info do
     end
   end
 
+  @doc """
+  Returns a list of extensions in use by the domain.
+
+  ## Options
+  - `:include_resource_extensions?` - If `true`, includes extensions defined on
+    resources in the domain. Defaults to `false`.
+  """
+  @spec extensions(domain :: Ash.Domain.t(), opts :: [include_resource_extensions?: boolean()]) ::
+          list(module())
+  def extensions(domain, opts \\ []) do
+    if opts[:include_resource_extensions?] do
+      domain
+      |> resources()
+      |> Enum.flat_map(&Ash.Resource.Info.extensions/1)
+      |> Enum.uniq()
```

Comment:
The `Enum.uniq` should be moved down. There are extensions that can go on bot domains and resources.